pyproject.toml

```diff
@@ -53,7 +53,7 @@ classifiers = [
 dependencies = [
     "absl-py>=2.0.0,<3.0.0",
     "comment_parser",
-    "cosmos-xenna==0.1.2",
+    "cosmos-xenna>=0.1.8",
```

Comment:
Do you prefer a `>=` pin here over an exact one? 

For release I would recommend an exact pin to ensure that users getting 26.02 don't end up pulling a newer version of xenna if it's incompatible.